kolibri_explore_plugin/kolibri_plugin.py

```diff
@@ -56,6 +56,14 @@ def plugin_data(self):
             ],
             "hideDiscoveryTab": conf.OPTIONS["Explore"]["HIDE_DISCOVERY_TAB"],
             "feedbackUrl": conf.OPTIONS["Explore"]["FEEDBACK_URL"],
+            # Use the app IDs from the PWA plugin, which advertises them as
+            # related applications.
+            "androidApplicationId": conf.OPTIONS["Pwa"][
+                "ANDROID_APPLICATION_ID"
+            ],
+            "windowsApplicationId": conf.OPTIONS["Pwa"][
+                "WINDOWS_APPLICATION_ID"
```

Comment:
Ah, this is the first time that a setting from another plugin is referenced in the kolibri-explore-plugin. This is what happens when the PWA plugin is not installed. Example from my local environment but it will happen in Android/Windows apps too:

![Captura de pantalla de 2023-06-21 16-59-16](https://github.com/endlessm/kolibri-explore-plugin/assets/83944/b46a258f-07d4-429a-8e4b-383bbd750c80)